decidim-verifications/spec/system/authorizations_spec.rb

```diff
@@ -40,6 +40,55 @@
         expect(page).to have_current_path decidim.account_path
         expect(page).to have_content("Participant settings")
       end
+
+      context "and a duplicate authorization exists for a deleted user" do
```

Comment:
I was thinking that we should add an example for the scenario where there's already an authorization for another user that wasn't deleted (and it doesn't transfer the authorization, of course). I can't seem to find it in the current examples: 

```bash
$ cd decidim-verifications && bundle exec rspec -f d ./spec/system/authorizations_spec.rb

(...) 

Authorizations
  when existing user from their account
    when no authorizations are configured
      doesn't list authorizations
    when user has not already been authorized
      allows the user to authorize against available authorizations
      checks if the given data is invalid
    when the user has already been authorized
      shows the authorization at their account
      when the authorization is renewable
        and passed the time between renewals
          can be renewed
          shows a modal with renew information
          and clicks on the button to renew
            shows the verification form to start again
        and still not over the waiting period
          can't be renewed yet
      when the authorization has not expired yet
        can't be renewed yet
      when the authorization has expired
        can be renewed
  when a new user
    when multiple authorizations have been configured
      allows the user to choose which one to authorize against to
    when one authorization has been configured
      allows the user to skip it
      redirects the user to the authorization form after the first sign in
      and a duplicate authorization exists for a deleted user
        transfers the authorization from the deleted user
        and the deleted user for the duplicate authorization had transferrable data
          reports the transferred participation data

Finished in 1 minute 35.9 seconds (files took 7.28 seconds to load)
15 examples, 0 failures

(...)
```
But seeing that we're checking that in [authorize_user_spec.rb](https://github.com/mainio/decidim/blob/91441a958064ec82ee66eecf7192bc64418c0c91/decidim-verifications/spec/commands/decidim/verifications/authorize_user_spec.rb#L78) and also in [authorizations_controller_spec.rb](https://github.com/mainio/decidim/blob/91441a958064ec82ee66eecf7192bc64418c0c91/decidim-verifications/spec/controllers/decidim/authorizations_controller_spec.rb#L47), I think that we can skip it.

Reply:
Added specs.